incubator/foundry-vtt/templates/deployment.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
 spec:
   strategy:
     type: Recreate
-  replicas: 1
+  replicas: {{ .Values.replicas | default 1 }}
```

Comment:
Hi @dirien I left the replica as only 1 as Foundry is 1 License per Game instance, to be compliant with the license of the software it self. 

The idea is to create 1 deploy for each instance.

Having multiple instances also causes issues with the in-game Websocket as there will be a instance of the game that the gamemaster is not logged so nothing will happen.

Reply:
We use a GitOps approach, and would to scale to zero via the values of the Helm chart. Modifying the deployment will not work due to the reconcile.